.github/workflows/main.yaml

```diff
@@ -0,0 +1,50 @@
+name: test
```

Comment:
Is this name `test` correctly?

Reply:
Isn't the pipeline name?

Reply:
this name is the one displayed in the generated badge to include in readme.md